types/api/threads.d.ts

```diff
@@ -11,7 +11,6 @@ type Thread = {
     is_following?: boolean;
     unread_replies: number;
     unread_mentions: number;
-    loaded_in_global_threads: boolean;
```

Comment:
Does it mean that the server is not sending this property?

Reply:
the server is not sending this property no, we were adding it to threads manually.
But it had been removed from there and added to THREADS_IN_TEAM table!